example/views.py

```diff
@@ -17,6 +20,8 @@ class BlogCustomViewSet(viewsets.ModelViewSet):
     serializer_class = BlogSerializer
 
     def handle_exception(self, exc):
+        if isinstance(exc, exceptions.ValidationError):
+            exc.status_code = HTTP_422_UNPROCESSABLE_ENTITY
```

Comment:
This is a good example for setting it manually. jsonapi.org says they don't take a stand on 400 vs 422 so the Ember addons should really support both in my opinion.